cgmanifest.json

```diff
@@ -4630,8 +4630,8 @@
         "type": "other",
         "other": {
           "name": "golang",
-          "version": "1.19.12",
-          "downloadUrl": "https://golang.org/dl/go1.19.12.src.tar.gz"
```

Comment:
Since this source is still used for the bootstrap, I think we should keep it here

Reply:
Thank you @anphel31.
I was in delima about the same issue for a different PR as well. Thank you for clarification. 
WIll do :)